Presentation/Fragments/ContactFragment.cs

```diff
@@ -0,0 +1,203 @@
+using System.IO;
+using Android.Graphics;
+
+namespace Presentation.Fragments
+{
+    public class ContactFragment : Fragment
+    {
+        private readonly SwaggerServices services = new SwaggerServices();
+
+        private View view;
+
+        [InjectView(Resource.Id.imgCard)] private ImageView imgCard;
+
+        [InjectView(Resource.Id.cbDigitalProject)] private CheckBox cbDigitalProject;
+
+        [InjectView(Resource.Id.cbJobOrMasterThesis)] private CheckBox cbJobOrMasterThesis;
+
+        [InjectView(Resource.Id.cbSayHi)] private CheckBox cbSayHi;
+
+        [InjectView(Resource.Id.cbOther)] private CheckBox cbOther;
+
+        [InjectView(Resource.Id.tvFirstName)] private EditText tvFirstName;
+
+        [InjectView(Resource.Id.tvLastName)] private EditText tvLastName;
+
+        [InjectView(Resource.Id.tvAddress)] private EditText tvAddress;
+
+        [InjectView(Resource.Id.tvCompany)] private EditText tvCompany;
+
+        [InjectView(Resource.Id.tvPosition)] private EditText tvPosition;
+
+        [InjectView(Resource.Id.tvBranch)] private EditText tvBranch;
+
+        [InjectView(Resource.Id.tvComments)] private EditText tvComments;
+
+        [InjectView(Resource.Id.btnSend)] private Button btnSend;
+
+        [InjectView(Resource.Id.spnTo)] private Spinner spnTo;
+
+        [InjectOnClick(Resource.Id.btnCamera)]
+        private void TakeImage(object sender, EventArgs e)
+        {
+            var imageIntent = new Intent();
+            imageIntent.SetType("image/*");
+            imageIntent.SetAction(Intent.ActionGetContent);
+            StartActivityForResult(Intent.CreateChooser(imageIntent, "Select photo"), 0);
+        }
+
+        [InjectOnClick(Resource.Id.btnSend)]
+        private void Send(object sender, EventArgs e)
+        {
+            Log.Info("string", imageBase64String);
+            var contact = new Contact
+            {
+                Reason = reason,
+                FirstName = tvFirstName.Text,
+                LastName = tvLastName.Text,
+                HelloTo = helloTo,
+                Email = tvAddress.Text,
+                Industry = tvBranch.Text,
+                Position = tvPosition.Text,
+                Token = "v5?1Q$Q974-tE_i1K!_!7qoiEo_?4@-35ptR57tedi66Mx-Duqm-2?x$j@zX6@U",
+                Company = tvCompany.Text,
+                Comments = tvComments.Text,
+                Images = string.IsNullOrEmpty(imageBase64String) ? new List<Image>() : new List<Image>
+                {
+                    new Image
+                    {
+                        Id = 0,
+                        ImageBase64 = imageBase64String
+                    }
+                }
+            };
+            if (services.PostContact(contact))
+                Toast.MakeText(view.Context, "Success", ToastLength.Short).Show();
+            else
+                Toast.MakeText(view.Context, "Failed", ToastLength.Short).Show();
+        }
+
+        private List<CheckBox> checkboxes;
```

Comment:
Please move those `fields` to the above of the first method.

Reply:
Done